tmt/steps/provision/testcloud.py

```diff
@@ -957,8 +962,17 @@ def start(self) -> None:
         os.makedirs(TESTCLOUD_DATA, exist_ok=True)
         os.makedirs(TESTCLOUD_IMAGES, exist_ok=True)
 
+        # Prepare the console log
+        assert self.workdir is not None  # Narrow type
+        console_log = ConsoleLog(
+            name=CONSOLE_LOG_FILE, symlink=self.workdir / CONSOLE_LOG_FILE, guest=self
```

Comment:
I'd argue `symlink` should be stored under `self.logdir` rather than `self.workdir`. it will be replaced by a regular file later, but as long as anyone peeks into `provision/$guestname/logs`, they should find the logs.